workerbase/lib/build_crosscompiler.sh

```diff
@@ -136,6 +143,8 @@ install_binutils()
 
     # Build binutils!
     cd /src/binutils-${binutils_version}
+    # NOTE: Fixed in binutils 2.29.1
+    patch -p0 < /downloads/patches/binutils-musl.patch
```

Comment:
If this is fixed in 2.29.1, do we still need the patch?

Reply:
Still set to 2.28 above. Didn't want to deal with yet another change, but we can bump it and remove the patch.

Reply:
What about https://github.com/staticfloat/julia-docker/pull/3/files#diff-e85a9af2d8ec9bcec813ba09d65e10f9R2

Reply:
Ah that's for the overall binutils. We still build a binutils per target.

Reply:
Gotcha.  Let's just bump up to `2.29.1` everywhere and have done with it.

Reply:
Ok